The patch is no longer needed for DF 3.7+, but still needed for DF3.6-

--- Makefile.orig	2014-02-09 20:45:37.281688000 +0000
+++ Makefile
@@ -98,8 +98,8 @@ post-build:
 	    | ${GREP} -v 'not a dynamic executable' \
 	    | ${GREP} '=>' \
 	    | ${AWK}  '{print $$3;}' \
-	    | ${EGREP} -v '^/lib/libgcc(_s)?\.so\.' \
-	    | ${GREP} -v '^/lib/libc\.so\.' || :)"; \
+	    | ${EGREP} -v 'lib/libgcc(_s)?\.so\.' \
+	    | ${GREP} -v 'lib/libc\.so\.' || :)"; \
 	if test "x$$a" = "x" ; then echo 'PASS' ; else \
 	    echo 'FAIL' ; echo '===>  e2fsck depends on:' ; echo "$$a" ; exit 1 ; fi
 # Update translation binary files
